packages/just-task/src/package/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from './findDependents';
```

Comment:
`workspace-tools` doesn't appear to have a function with this exact name, but it should have things with similar functionality--try `getDependentMap` or one of the other functions from here maybe? https://github.com/microsoft/workspace-tools/blob/master/src/dependencies.ts

Reply:
Thanks! I'll take a look!